packages/aws-cdk/lib/cli.ts

```diff
@@ -90,6 +90,7 @@ async function parseCommandLineArguments() {
     .command('bootstrap [ENVIRONMENTS..]', 'Deploys the CDK toolkit stack into an AWS environment', (yargs: Argv) => yargs
       .option('bootstrap-bucket-name', { type: 'string', alias: ['b', 'toolkit-bucket-name'], desc: 'The name of the CDK toolkit bucket; bucket will be created and must not exist', default: undefined })
       .option('bootstrap-kms-key-id', { type: 'string', desc: 'AWS KMS master key ID used for the SSE-KMS encryption', default: undefined, conflicts: 'bootstrap-customer-key' })
+      .option('bootstrap-ecr-key', { type: 'string', desc: 'Encrypts the ecr with an Customer Master Key (CMK) or with the key given by an optional key-Id', conflicts: 'bootstrap-customer-key' })
```

Comment:
ERC is an AWS product so it should be stylized correctly here. I'm also not super happy with the lack of consistency in our three bootstrap-key command names, but I'm not totally sure what a better on would be in this case.

Reply:
After making this comment, I realize that we don't do the same with kms here so the stylizing comment can be ignored.